packages/ilib-loctool-mrkdwn/test/MrkdwnJsFile.test.js

```diff
@@ -791,6 +804,37 @@ describe("mrkdwn", function() {
         expect(actual).toBe(expected);
     });
 
+    test("MrkdwnJsFile output commonjs format file", function() {
+        expect.assertions(2);
+        var mjf = new MrkdwnJsFile({
+            project: p4,
+            type: mdft4
+        });
+
+        expect(mjf).toBeTruthy();
+        mjf.parse(
```

Comment:
What about using string literal syntax here? It allows the string to expand into multiple lines without `\n` and hence is more readable
```suggestion
        mjf.parse(`
    export default messages = {
        "id1": "This is a test",
    };
`);
```

Reply:
This plugin is written using commonjs/ES2015 at the moment and is not transpiled. It would not make sense to start introducing ESM features here.